docs/en/writing-running-appium/tutorial/swipe/android-simple.md

```diff
@@ -0,0 +1,98 @@
+## Android 'UIScrollable' swipe: Simple example
+
+UIScrollable is powerful Android class that searching elements in
```

Comment:
...is a powerful...
...that performs element lookups in scrollable layouts..
...until the destination element is found on the screen.